[2017-11-01 22:26:23] Shadowsocks started
[2017-11-01 22:27:00] Shadowsocks started
[2017-11-01 22:27:04] Shadowsocks started
[2017-11-01 22:27:30] Shadowsocks started
[2017-11-01 22:27:51] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:12] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:31] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:38] Shadowsocks started
[2017-11-01 22:28:45] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2017-11-01 22:28:49] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:49] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:49] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:51] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:53] Shadowsocks started
[2017-11-01 22:28:58] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2017-11-01 22:28:58] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2017-11-01 22:28:59] miao (138.68.21.244:443) timed out
[2017-11-01 22:28:59] miao (138.68.21.244:443) timed out
[2017-11-01 22:29:00] miao (138.68.21.244:443) timed out
[2017-11-01 22:29:02] miao (138.68.21.244:443) timed out
[2017-11-01 22:29:04] miao (138.68.21.244:443) timed out
[2017-11-01 22:29:05] miao (138.68.21.244:443) timed out
[2017-11-01 22:29:14] Shadowsocks started
[2017-11-01 22:29:25] HA switching to server: miao (138.68.21.244:443)
[2017-11-01 22:29:25] Shadowsocks started
[2017-11-01 22:29:27] HA switching to server: 未配置的服务器
[2017-11-01 22:29:27] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:27] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:27] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:28] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:28] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:31] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:32] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-01 22:29:33] Shadowsocks started
[2017-11-01 22:33:53] Shadowsocks started
[2017-11-01 22:35:12] Shadowsocks started
[2017-11-01 22:52:07] miao (138.68.21.244:443) timed out
[2017-11-01 22:53:13] miao (138.68.21.244:443) timed out
[2017-11-01 22:53:50] miao (138.68.21.244:443) timed out
[2017-11-01 22:54:05] miao (138.68.21.244:443) timed out
[2017-11-01 22:54:05] miao (138.68.21.244:443) timed out
[2017-11-01 22:54:09] miao (138.68.21.244:443) timed out
[2017-11-01 22:54:14] miao (138.68.21.244:443) timed out
[2017-11-01 23:00:07] miao (138.68.21.244:443) timed out
[2017-11-01 23:00:07] miao (138.68.21.244:443) timed out
[2017-11-01 23:00:17] miao (138.68.21.244:443) timed out
[2017-11-01 23:04:32] miao (138.68.21.244:443) timed out
[2017-11-01 23:33:47] controller stopped
[2017-11-01 23:33:47] os suspend
[2017-11-01 23:33:59] os wake up
[2017-11-01 23:34:10] Shadowsocks started
[2017-11-01 23:34:10] controller started
[2017-11-01 23:40:00] miao (138.68.21.244:443) timed out
[2017-11-01 23:40:07] miao (138.68.21.244:443) timed out
[2017-11-01 23:40:15] miao (138.68.21.244:443) timed out
[2017-11-01 23:40:15] miao (138.68.21.244:443) timed out
[2017-11-01 23:40:28] miao (138.68.21.244:443) timed out
[2017-11-01 23:40:40] miao (138.68.21.244:443) timed out
[2017-11-01 23:41:04] miao (138.68.21.244:443) timed out
[2017-11-01 23:41:11] miao (138.68.21.244:443) timed out
[2017-11-01 23:41:21] miao (138.68.21.244:443) timed out
[2017-11-01 23:41:31] miao (138.68.21.244:443) timed out
[2017-11-01 23:42:30] miao (138.68.21.244:443) timed out
[2017-11-01 23:42:37] miao (138.68.21.244:443) timed out
[2017-11-01 23:42:59] miao (138.68.21.244:443) timed out
[2017-11-01 23:43:13] miao (138.68.21.244:443) timed out
[2017-11-01 23:43:13] miao (138.68.21.244:443) timed out
[2017-11-01 23:43:21] miao (138.68.21.244:443) timed out
[2017-11-02 19:53:41] Shadowsocks started
[2017-11-02 19:53:47] miao (138.68.21.244:443) timed out
[2017-11-02 19:53:49] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 基础连接已经关闭: 发送时发生错误。 ---> System.IO.IOException: 由于远程方已关闭传输流，身份验证失败。
   在 System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   在 System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   在 System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2017-11-02 19:53:49] Shadowsocks started
[2017-11-02 19:54:01] Shadowsocks started
[2017-11-02 19:56:40] Shadowsocks started
[2017-11-02 19:56:40] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-02 19:56:40] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-02 19:56:40] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 基础连接已经关闭: 发送时发生错误。 ---> System.IO.IOException: 由于远程方已关闭传输流，身份验证失败。
   在 System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   在 System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   在 System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2017-11-02 19:56:46] Shadowsocks started
[2017-11-02 20:00:09] Shadowsocks started
[2017-11-02 20:00:15] Shadowsocks started
[2017-11-02 20:00:15] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-02 20:00:16] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 基础连接已经关闭: 发送时发生错误。 ---> System.IO.IOException: 由于远程方已关闭传输流，身份验证失败。
   在 System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   在 System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   在 System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2017-11-10 22:24:26] Shadowsocks started
[2017-11-10 22:24:28] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-10 22:24:28] System.ArgumentException: No server configured
   在 Shadowsocks.Controller.TCPHandler.CreateRemote()
   在 Shadowsocks.Controller.TCPHandler.StartConnect()
[2017-11-10 22:24:28] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 基础连接已经关闭: 发送时发生错误。 ---> System.IO.IOException: 由于远程方已关闭传输流，身份验证失败。
   在 System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   在 System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   在 System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2017-11-10 22:24:33] Shadowsocks started
[2017-11-10 22:24:35] Shadowsocks started
[2017-11-11 11:13:25] Shadowsocks started
[2017-11-11 11:59:35] controller stopped
[2017-11-11 11:59:35] os suspend
[2017-11-11 21:42:37] os wake up
[2017-11-11 21:42:38] controller stopped
[2017-11-11 21:42:38] os suspend
[2017-11-11 21:42:49] Shadowsocks started
[2017-11-11 21:42:50] controller started
[2017-11-11 21:42:55] os wake up
[2017-11-11 21:50:31] Shadowsocks started
[2017-11-11 21:59:04] Shadowsocks started
[2017-11-11 22:33:56] Shadowsocks started
[2017-12-04 16:34:53] Shadowsocks started
[2017-12-04 16:34:57] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:34:58] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:34:59] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:34:59] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 基础连接已经关闭: 发送时发生错误。 ---> System.IO.IOException: 由于远程方已关闭传输流，身份验证失败。
   在 System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)
   在 System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   在 System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   在 System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2017-12-04 16:35:00] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:00] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:12] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:27] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:28] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:30] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:30] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:32] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:34] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-04 16:35:40] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Proxy.DirectConnect.EndConnectDest(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.TCPHandler.ConnectCallback(IAsyncResult ar)
[2017-12-09 13:56:05] Shadowsocks started
[2017-12-09 13:56:09] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 在 WebClient 请求期间发生异常。 ---> System.IO.IOException: 从传输流收到意外的 EOF 或 0 个字节。
   在 System.Net.ConnectStream.EndRead(IAsyncResult asyncResult)
   在 System.Net.WebClient.DownloadBitsReadCallbackState(DownloadBitsState state, IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2017-12-09 13:56:09] Shadowsocks started
[2017-12-09 14:22:37] Shadowsocks started
[2017-12-09 14:22:42] Shadowsocks started
[2017-12-09 14:28:36] Shadowsocks started
[2017-12-09 22:05:15] Shadowsocks started
[2017-12-09 22:05:19] Shadowsocks started
[2017-12-09 22:05:37] miao (138.68.21.244:443) timed out
[2017-12-09 22:06:28] miao (138.68.21.244:443) timed out
[2017-12-09 22:06:51] miao (138.68.21.244:443) timed out
[2017-12-09 22:06:52] miao (138.68.21.244:443) timed out
[2017-12-09 22:06:57] miao (138.68.21.244:443) timed out
[2017-12-09 22:06:58] miao (138.68.21.244:443) timed out
[2017-12-09 22:07:08] miao (138.68.21.244:443) timed out
[2017-12-09 22:07:31] miao (138.68.21.244:443) timed out
[2017-12-09 22:07:34] Shadowsocks started
[2017-12-09 22:07:36] Shadowsocks started
[2017-12-09 22:07:41] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2017-12-09 22:07:45] miao (138.68.21.244:443) timed out
[2017-12-09 22:08:06] Shadowsocks started
[2017-12-09 22:08:28] miao (138.68.21.244:443) timed out
[2017-12-09 22:08:28] miao (138.68.21.244:443) timed out
[2017-12-09 22:08:30] miao (138.68.21.244:443) timed out
[2017-12-09 22:08:34] miao (138.68.21.244:443) timed out
[2017-12-09 22:10:55] Shadowsocks started
[2017-12-09 22:10:59] Shadowsocks started
[2017-12-09 23:06:08] Shadowsocks started
[2017-12-10 09:20:40] Shadowsocks started
[2017-12-10 09:21:23] Shadowsocks started
[2017-12-10 10:46:02] Shadowsocks started
[2017-12-10 19:23:07] Shadowsocks started
[2017-12-10 19:23:35] Shadowsocks started
[2017-12-10 19:24:03] Shadowsocks started
[2017-12-10 19:35:49] controller stopped
[2017-12-10 19:35:49] os suspend
[2017-12-10 20:06:30] os wake up
[2017-12-10 20:06:40] Shadowsocks started
[2017-12-10 20:06:41] controller started
[2017-12-15 23:23:19] Shadowsocks started
[2017-12-15 23:23:39] Shadowsocks started
[2017-12-15 23:45:11] Shadowsocks started
[2017-12-15 23:45:38] Shadowsocks started
[2017-12-15 23:45:41] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2017-12-15 23:49:23] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:25] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:27] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:28] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:30] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2017-12-15 23:49:30] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:32] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:33] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:36] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:36] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:36] jp.ohmyss.win:30800 timed out
[2017-12-15 23:49:37] jp.ohmyss.win:30800 timed out
[2017-12-15 23:51:01] controller stopped
[2017-12-15 23:51:01] os suspend
[2017-12-16 08:36:12] os wake up
[2017-12-16 08:36:23] Shadowsocks started
[2017-12-16 08:36:23] controller started
[2017-12-16 08:36:26] controller stopped
[2017-12-16 08:36:26] os suspend
[2017-12-17 20:57:56] Shadowsocks started
[2017-12-17 21:01:20] jp.ohmyss.win:30800 timed out
[2017-12-17 21:01:38] jp.ohmyss.win:30800 timed out
[2017-12-17 21:30:22] jp.ohmyss.win:30800 timed out
[2017-12-17 21:30:28] jp.ohmyss.win:30800 timed out
[2017-12-17 21:33:04] Shadowsocks started
[2017-12-25 21:30:07] Shadowsocks started
[2017-12-25 21:40:41] Shadowsocks started
[2017-12-25 21:53:49] Shadowsocks started
[2018-01-01 15:36:11] Shadowsocks started
[2018-01-01 15:36:18] Shadowsocks started
[2018-01-01 15:37:14] Shadowsocks started
[2018-01-01 15:37:20] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:22] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:25] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:27] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:27] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:27] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:27] miao (138.68.21.244:443) timed out
[2018-01-01 15:37:36] Shadowsocks started
[2018-01-01 15:52:04] jp.ohmyss.win:30800 timed out
[2018-01-01 15:59:17] Shadowsocks started
[2018-01-01 16:05:42] Shadowsocks started
[2018-01-01 16:23:09] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2018-01-01 16:33:36] controller stopped
[2018-01-01 16:33:36] os suspend
[2018-01-01 16:39:16] os wake up
[2018-01-01 16:39:26] Shadowsocks started
[2018-01-01 16:39:27] Shadowsocks started
[2018-01-08 19:23:13] Shadowsocks started
[2018-01-08 19:23:16] Shadowsocks started
[2018-01-08 19:39:37] Shadowsocks started
[2018-01-20 19:42:07] Shadowsocks started
[2018-01-20 19:42:57] Shadowsocks started
[2018-01-20 19:48:26] controller stopped
[2018-01-20 19:48:26] os suspend
[2018-01-20 19:50:22] os wake up
[2018-01-20 19:50:32] Shadowsocks started
[2018-01-20 19:50:33] controller started
[2018-01-20 19:52:28] controller stopped
[2018-01-20 19:52:28] os suspend
[2018-01-26 23:04:11] Shadowsocks started
[2018-01-26 23:06:19] jp.ohmyss.win:30800 timed out
[2018-01-26 23:07:10] jp.ohmyss.win:30800 timed out
[2018-01-26 23:09:34] Shadowsocks started
[2018-01-26 23:16:42] Shadowsocks started
[2018-01-26 23:17:04] System.Net.WebException: 在 WebClient 请求期间发生异常。 ---> System.IO.IOException: 从传输流收到意外的 EOF 或 0 个字节。
   在 System.Net.ConnectStream.EndRead(IAsyncResult asyncResult)
   在 System.Net.WebClient.DownloadBitsReadCallbackState(DownloadBitsState state, IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
[2018-01-26 23:17:04] Shadowsocks started
[2018-01-26 23:20:38] Shadowsocks started
[2018-01-26 23:20:43] Shadowsocks started
[2018-01-26 23:21:05] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2018-01-26 23:28:12] Shadowsocks started
[2018-01-26 23:28:18] Shadowsocks started
[2018-01-26 23:28:45] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2018-01-26 23:33:39] Shadowsocks started
[2018-01-26 23:33:44] Shadowsocks started
[2018-01-26 23:34:08] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2018-01-26 23:34:26] Shadowsocks started
[2018-01-26 23:34:32] Shadowsocks started
[2018-01-26 23:34:52] Shadowsocks started
[2018-02-01 19:33:07] Shadowsocks started
[2018-02-01 19:33:11] System.Reflection.TargetInvocationException: 操作过程中出现异常，结果无效。  有关异常的详细信息，请查看 InnerException。 ---> System.Net.WebException: 在 WebClient 请求期间发生异常。 ---> System.IO.IOException: 从传输流收到意外的 EOF 或 0 个字节。
   在 System.Net.ConnectStream.EndRead(IAsyncResult asyncResult)
   在 System.Net.WebClient.DownloadBitsReadCallbackState(DownloadBitsState state, IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()
   在 Shadowsocks.Controller.UpdateChecker.http_DownloadStringCompleted(Object sender, DownloadStringCompletedEventArgs e)
[2018-02-01 19:33:11] Shadowsocks started
[2018-02-01 19:42:30] Shadowsocks started
[2018-02-01 20:42:55] Shadowsocks started
[2018-02-01 20:43:42] System.Net.WebException: 在 WebClient 请求期间发生异常。 ---> System.IO.IOException: 从传输流收到意外的 EOF 或 0 个字节。
   在 System.Net.ConnectStream.EndRead(IAsyncResult asyncResult)
   在 System.Net.WebClient.DownloadBitsReadCallbackState(DownloadBitsState state, IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
[2018-02-01 20:43:42] Shadowsocks started
[2018-02-01 21:06:42] Shadowsocks started
[2018-02-03 09:17:58] Shadowsocks started
[2018-02-03 09:18:00] Shadowsocks started
[2018-02-03 09:20:08] System.Net.Sockets.SocketException (0x80004005): 由于以前的关闭调用，套接字在那个方向已经关闭，发送或接收数据的请求没有被接受。
   在 System.Net.Sockets.Socket.BeginSend(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, AsyncCallback callback, Object state)
   在 Shadowsocks.Controller.TCPHandler.PipeRemoteReceiveCallback(IAsyncResult ar)
[2018-02-03 09:20:09] controller stopped
[2018-02-03 09:20:09] os suspend
[2018-02-03 09:20:30] os wake up
[2018-02-03 09:20:41] Shadowsocks started
[2018-02-03 09:20:42] controller started
[2018-02-03 09:43:33] controller stopped
[2018-02-03 09:43:33] os suspend
[2018-02-03 09:46:58] os wake up
[2018-02-03 09:47:09] Shadowsocks started
[2018-02-03 09:47:09] controller started
[2018-02-03 10:48:23] Shadowsocks started
[2018-02-03 18:54:56] Shadowsocks started
[2018-02-03 18:55:05] System.Net.WebException: 在 WebClient 请求期间发生异常。 ---> System.IO.IOException: 从传输流收到意外的 EOF 或 0 个字节。
   在 System.Net.ConnectStream.EndRead(IAsyncResult asyncResult)
   在 System.Net.WebClient.DownloadBitsReadCallbackState(DownloadBitsState state, IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
[2018-02-03 18:55:05] Shadowsocks started
[2018-02-03 19:29:19] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:01] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:06] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:19] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:21] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:21] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:32] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:39] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:42] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:55] jp.ohmyss.win:30800 timed out
[2018-02-03 19:39:58] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:00] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:05] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:21] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:22] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:25] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:25] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:25] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:26] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:31] Shadowsocks started
[2018-02-03 19:40:33] Shadowsocks started
[2018-02-03 19:40:35] System.Net.Sockets.SocketException (0x80004005): 由于目标计算机积极拒绝，无法连接。
   在 Shadowsocks.Util.Sockets.WrappedSocket.EndConnect(IAsyncResult asyncResult)
   在 Shadowsocks.Controller.PortForwarder.Handler.ConnectCallback(IAsyncResult ar)
[2018-02-03 19:40:41] jp.ohmyss.win:30800 timed out
[2018-02-03 19:40:48] Shadowsocks started
[2018-02-03 19:55:03] Shadowsocks started
[2018-02-03 19:55:05] Shadowsocks started
[2018-02-03 20:15:46] Shadowsocks started
[2018-02-04 16:08:02] Shadowsocks started
[2018-02-04 16:08:12] System.Net.WebException: 在 WebClient 请求期间发生异常。 ---> System.IO.IOException: 从传输流收到意外的 EOF 或 0 个字节。
   在 System.Net.ConnectStream.EndRead(IAsyncResult asyncResult)
   在 System.Net.WebClient.DownloadBitsReadCallbackState(DownloadBitsState state, IAsyncResult result)
   --- 内部异常堆栈跟踪的结尾 ---
